src/js/date.js

```diff
@@ -95,3 +96,58 @@ export const getNumericWeekday = (locale = 'en-uk', date = new Date()) => {
   });
   return weekdayIndex;
 };
+
+export const getLocaleDayMonthYear = (locale = 'en-uk', date = new Date()) => {
+  const objDate = date.toLocaleString(locale, { day: 'numeric', month: 'numeric', year: 'numeric' });
+  return objDate;
+};
+
+export const isDateValid = (locale = 'en-uk', inputValue = '') => {
+  let splittedInput = [];
+  let orderedInput = [];
+  let date = null;
+
+  if (locale === 'de-ch') {
+    splittedInput = inputValue.split('.');
+    orderedInput = [splittedInput[1], splittedInput[0], splittedInput[2]];
+    date = new Date(orderedInput.join('.'));
+  } else {
+    date = new Date(inputValue);
+  }
+
+  let objDate = '';
+  const numeric = 'numeric';
+  const twoDigit = '2-digit';
+  const selectors = [numeric, twoDigit];
+  const filters = ['year', 'month', 'day'];
+  let options = {};
+  let valid = false;
+
+  filters.forEach((filter) => {
+    options[filter] = '';
+  });
+
+  const combinations = [];
+  selectors.forEach((el1) => {
+    selectors.forEach((el2) => {
+      selectors.forEach((el3) => {
+        options = ({
+          year: el1,
```

Comment:
This seems to be overkill and those variable names aren't semantic - what is `el1`, `el2`...